### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by another-rex**
Update and improve readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,54 @@
...
-# OSV-Scanner (Preview)
+# OSV-Scanner
 
-OSV-Scanner is a vulnerability scanner written in Go which uses the data provided by https://osv.dev
+Use OSV-Scanner to find existing vulnerabilities affecting your project's dependencies.
 
-This tool is currently under development and is subject to change.
+OSV-Scanner provides an officially supported frontend to the [OSV database](https://osv.dev/) that connects a project’s list of dependencies with the vulnerabilities that affect them. Since the OSV.dev database is open source and distributed, it has several benefits in comparison with closed source advisory databases and scanners:
+
+- Each advisory comes from an open and authoritative source (e.g. the RustSec Advisory Database)
```
Comment (oliverchang) [inline README.md:9]: nit: link to the RustSec advisory DB.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
 
 ## Installing
 
+### Prerequisites
+Requires go 1.18+ to be installed
+
+### Installation Process
+Run
 ```bash
 $ go install github.com/google/osv-scanner/cmd/osv-scanner@latest
```
Comment (oliverchang) [inline README.md:23]: Should we also add a section on how we will adhere to SemVer and what guarantees there are? 

Also, we'll probably want to replace this @latest with v1 or something too.

**Review by oliverchang — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -44,57 +80,33 @@ A wide range of lockfiles are supported by utilizing this [lockfile package](htt
...
-By placing a `osv-scanner.toml` file in any parent directory of the file being
-scanned will be used to configure scanning of that file. This can be overridden
-by passing the `--config=/path/to/config` flag.
+To configure scanning, place an osv-scanner.toml file in the scanned file's directory. To override this osv-scanner.toml file, pass the `--config=/path/to/config.toml` flag with the path to the configuration you want to apply instead.
 
 Currently, there is only 1 option to configure:
 ### Ignore vulnerabilities by ID
-Vulnerabilities can be marked as ignored by putting the ID an entry
-under the `IgnoreVulns` key, along with optional reason and expiry date.
+To ignore a vulnerability, enter the ID under the `IgnoreVulns` key. Optionally, add an expiry date or reason.
```
Comment (oliverchang) [inline README.md:109]: Also asked this earlier today but wanted to clarify again:

If we have the following directory structure:

```
foo/osv-scanner.toml
foo/package-lock.json
bar/osv-scanner.toml
bar/package-lock.json
osv-scanner.toml
package-lock.json
```

Will scanning:

- `foo/package-lock.json` use `foo/osv-scanner.toml` by default? 
- `bar/package-lock.json` use `bar/osv-scanner.toml` by default? 
- `package-lock.json` use `osv-scanner.toml` by default?

Comment (another-rex) [reply README.md:109]: Yep, that's correct

File: README.md (hunk 2 above)
Comment (another-rex) [reply README.md]: changed this to v1, I'll push a tag after this PR merged, which I think is all that's needed to create a version?

Also added a line about semver adherence.

**Pull request merged**